### PR TITLE
gncosta updating caption

### PR DESCRIPTION
**Pull request by gncosta-gh**
gncosta updating caption

### CODE REVIEW

**Comment by gncosta-gh** [comment]
gncosta updating caption comment

File: _slides/09-gncosta-gh.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+gncosta-gh caption
```
Comment (gncosta-gh) [inline _slides/09-gncosta-gh.md:6]: change looks great

**Review by gncosta-gh — COMMENTED** [review]
approved

File: _slides/09-gncosta-gh.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ layout: slide
...
```
Comment (miguel-angelo-conceicao) [inline _slides/09-gncosta-gh.md:7]: Looks fine!

**Review by miguel-angelo-conceicao — COMMENTED** [review]
OK!

**Pull request merged**